### PR TITLE
feat: improve kotlin compiler speed and upgrade compiler

### PR DESCRIPTION
**Pull request by PranavPurwar**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
-<p align="center">
-  <img src="assets/Sketchware-Pro.png" />
+<p>
+  <img src="assets/Sketchware-Pro.png"  alt="Sketchware Pro logo"/>
 </p>
+<style>
+p {
+    text-align: center;
+}
+</style>
```
Comment (JavkhlanK) [inline README.md:8]: this breaks the centered image [even though the previous approach is the only one that works](https://stackoverflow.com/a/62383408)

File: kotlinc/src/main/java/org/jetbrains/kotlin/cli/jvm/compiler/jarfs/FastJarFileSystem.kt [hunk 2]
```diff
@@ -0,0 +1,131 @@
+/*
+ * This file is part of Cosmic IDE.
```
Comment (JavkhlanK) [inline kotlinc/src/main/java/org/jetbrains/kotlin/cli/jvm/compiler/jarfs/FastJarFileSystem.kt:2]: didn't you remove the original license and replace it with Cosmic IDE's?

**Review by JavkhlanK — CHANGES_REQUESTED** [review]

File: kotlinc/src/main/java/org/jetbrains/kotlin/cli/jvm/compiler/jarfs/FastJarFileSystem.kt (hunk 2 above)
Comment (PranavPurwar) [reply kotlinc/src/main/java/org/jetbrains/kotlin/cli/jvm/compiler/jarfs/FastJarFileSystem.kt:2]: ~~copied the module from cosmic ide, it was auto edited/added by AS~~

Comment (PranavPurwar) [reply kotlinc/src/main/java/org/jetbrains/kotlin/cli/jvm/compiler/jarfs/FastJarFileSystem.kt:2]: restored the original license header on files that haven't been modified from source

File: README.md (hunk 1 above)
Comment (PranavPurwar) [reply README.md]: fixed

**Pull request merged**